packages/design-system/src/components/OcTextInput/OcTextInput.spec.ts

```diff
@@ -52,6 +60,51 @@ describe('OcTextInput', () => {
     })
   })
 
+  describe('password input field', () => {
+    describe('copy password button', () => {
+      it('should not exist if type is not "password" or no value entered', () => {
+        const wrapper = getMountedWrapper()
+        expect(wrapper.find(selectors.copyPasswordBtn).exists()).toBeFalsy()
+
+        const wrapper2 = getMountedWrapper({ props: { type: 'password' } })
+        expect(wrapper2.find(selectors.copyPasswordBtn).exists()).toBeFalsy()
+      })
+      it('should exist if type is "password" and value entered', async () => {
+        const wrapper = getMountedWrapper({ props: { type: 'password' } })
+        await wrapper.find(selectors.inputField).setValue('password')
+        expect(wrapper.find(selectors.copyPasswordBtn).exists()).toBeTruthy()
+      })
+      it('should copy password to password if clicked', async () => {
```

Comment:
find the name of the test confusing, maybe 'should copy password to clipboard if clicked'